### PR TITLE
Feature template notes field

### PR DESCRIPTION
**Pull request by joshcbee**
Implementing staff #20 and student notes #21. 
- Modified globalRequirements and rules container to Div to allow repositioning
- Added studentNotes and staffNotes fields to create, edit, view as applicable, blank on create, persists from edit.
- modified Program Model with new fields.

Acceptance criteria:
- Users can enter staff comments in a template field.
- Users can view staff comments in a template field.
- Users can enter student comments in a template field.
- Text will be included as part of the printed template.

image

### CODE REVIEW

File: cassdegrees/templates/createprogram.html [hunk 1]
```diff
@@ -29,26 +29,54 @@
...
+                {{ form.staffNotes.label_tag }}
+                {{ form.staffNotes }}
+            </p>
+            {% for error in form.staffNotes.errors %}
+                <div class="msg-error inline-error">{{ error }}</div>
+            {% endfor %}
+        </fieldset>
     </form>
 
+    <!-- We still need a form for regular styling, though these will be serialized manually because of their list-like
```
Comment (j-selby) [inline cassdegrees/templates/createprogram.html]: This comment was attached to the above div

**Review by j-selby — APPROVED** [review]
Looks good!

**Review by JackSekoranja — APPROVED** [review]
Looks good to me, happy for you to merge

**Review by Daniel-Jang — APPROVED** [review]
Works well.

**Pull request merged**